serverless-hello-world/func.yaml

```diff
@@ -0,0 +1,20 @@
+name: serverless-hello-world
+namespace: ""
+runtime: node
+image: quay.io/somishra95/serverless-hello-world:latest
```

Comment:
At some point we need to move this image to the RedHatTraining namespace.

Reply:
I will remove the image name. That got here by mistake. By default it should ask for the name of image when you build it for first time.